integrations/lib/plugindata/access_request.go

```diff
@@ -63,6 +65,15 @@ func DecodeAccessRequestData(dataMap map[string]string) (data AccessRequestData,
 	}
 	data.ResolutionTag = ResolutionTag(dataMap["resolution"])
 	data.ResolutionReason = dataMap["resolve_reason"]
+	if str := dataMap["max_duration"]; str != "" {
+		var maxDuration time.Time
+		maxDuration, err = time.Parse(time.RFC3339, str)
+		if err != nil {
+			err = trace.Wrap(err)
+			return
```

Comment:
```suggestion
			return trace.Wrap(err)
```

This does exactly the same

Reply:
I don't think it is the same since we have the named returns `(data AccessRequestData, err error) `